src/bundle/Controller/SearchController.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace Ibexa\Platform\Bundle\SearchBundle\Controller;
+
+use eZ\Publish\API\Repository\ContentTypeService;
+use eZ\Publish\API\Repository\SearchService;
+use eZ\Publish\API\Repository\SectionService;
+use eZ\Publish\API\Repository\Values\Content\Language;
+use eZ\Publish\Core\MVC\ConfigResolverInterface;
+use eZ\Publish\Core\Pagination\Pagerfanta\ContentSearchHitAdapter;
+use eZ\Publish\Core\QueryType\QueryType;
+use EzSystems\EzPlatformAdminUi\Form\Data\Search\SearchData;
```

Comment:
maybe we want to "copy" SearchData, SearchType, and other similar classes to this package and deprecate original classes in Admin UI. Or lets remove them now from Admin UI